lib/tls.js

```diff
@@ -189,8 +183,7 @@ exports.checkServerIdentity = function checkServerIdentity(host, cert) {
 
   if (!valid) {
     var err = new Error(
-        util.format('Hostname/IP doesn\'t match certificate\'s altnames: %j',
-                    reason));
+        `Hostname/IP doesn't match certificate's altnames: "${reason}"`);
```

Comment:
I was assuming it was always a string but didn't dig through the code (best to dummy check :D)